dataikuapi/govern/admin_blueprint_designer.py

```diff
@@ -69,53 +58,47 @@ def __init__(self, client, blueprint_id):
+        return GovernAdminBlueprintDefinition(self.client, self.blueprint_id, blueprint["blueprint"])
 
     def list_versions(self, as_objects=True):
         """
         Lists versions of this blueprint.
 
         :param boolean as_objects: (Optional) If True, returns a list of :class:`~dataikuapi.govern.admin_blueprint_designer.GovernAdminBlueprintVersion`,
-         else returns a list of dict. Each dict contains a field "id.versionId" indicating the identifier of this version
+         else returns a list of dict. Each dict contains a field "blueprintVersion.id.versionId" indicating the identifier of this version
         :returns: The list of the versions of the blueprint
-        :rtype: list of :class:`~dataikuapi.govern.admin_blueprint_designer.GovernAdminBlueprintVersion` or list of dict
+        :rtype: list of :class:`~dataikuapi.govern.admin_blueprint_designer.GovernAdminBlueprintVersion` or list of dict, see parameter as_objects
         """
-        versions = self.client._perform_json(
-            "GET", "/admin/blueprint/%s/versions" % self.blueprint_id)
+        versions = self.client._perform_json("GET", "/admin/blueprint/%s/versions" % self.blueprint_id)
         if as_objects:
-            return [GovernAdminBlueprintVersion(self.client, self.blueprint_id, version["id"]["versionId"]) for version
-                    in versions]
+            return [GovernAdminBlueprintVersion(self.client, self.blueprint_id, version["blueprintVersion"]["id"]["versionId"]) for version in versions]
         else:
             return versions
 
     def create_version(self, new_identifier, blueprint_version_definition, origin_version_id=None):
         """
         Create a new blueprint version and returns a handle to interact with it.
 
-        :param str new_identifier: The new identifier of the blueprint version. This parameter should be made up of
-        letters, digits or hyphen (-,_).
+        :param str new_identifier: The new identifier of the blueprint version.
         :param dict blueprint_version_definition: The definition of the blueprint version.
         :param str origin_version_id: (Optional) The blueprint version id of the origin version id if there is one.
-        :returns The handle of the newly created blueprint
-        :rtype: :class:`dataikuapi.govern.admin_blueprint_designer.GovernAdminBlueprintVersion`
+        :returns: The handle of the newly created blueprint
+        :rtype: :class:`~dataikuapi.govern.admin_blueprint_designer.GovernAdminBlueprintVersion`
         """
         params = {"newIdentifier": new_identifier}
         if origin_version_id is not None:
             params["originVersionId"] = origin_version_id
 
-        blueprint_version_definition["id"] = {"blueprintId": self.blueprint_id}
+        version = self.client._perform_json("POST", "/admin/blueprint/%s/versions" % self.blueprint_id, params=params, body=blueprint_version_definition)
```

Comment:
Here we still need to add the `blueprintId` as an `id` key because the controller deserialize as a `BlueprintVersion`